### ISSUE TITLE
Remove gulp-util

### ISSUE THREAD

**Issue by unlight** [Dragory/gulp-hash#26]
I run into error with gulp 4.
```
TypeError: file.pipe is not a function (gulp-hash\index.js:30:20)
```
Solution:
https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

**Comment by unlight** [comment]
Who is facing same or [this issue](https://github.com/gulpjs/gulp/issues/2065)
Here is some details:
Looks like this commit broke it https://github.com/gulpjs/gulp/commit/99bdc9ead25ff02dd2309991eea3ebdb60ab8080

How to fix:
Rollback to `gulp@4.0.0-alpha.2`
Install `npm i -D gulpjs/gulp#6d71a658c61edb3090221579d8f97dbe086ba2ed`

**Comment by Dragory** [comment]
Thanks, I'll look into this

**Comment by Dragory** [comment]
Fixed by #27
